CHANGELOG.md

```diff
@@ -153,6 +153,11 @@ By @atlv24 in [#5383](https://github.com/gfx-rs/wgpu/pull/5383)
 - `wgpu::ComputePass` now internally takes ownership of `QuerySet` for both `wgpu::ComputePassTimestampWrites` as well as timestamp writes and statistics query, fixing crashes when destroying `QuerySet` before ending the pass. By @wumpf in [#5671](https://github.com/gfx-rs/wgpu/pull/5671)
 - Validate resources passed during compute pass recording for mismatching device. By @wumpf in [#5779](https://github.com/gfx-rs/wgpu/pull/5779)
 
+#### DX12
+
+- Do not feed `&""` to `D3DCompile`, by @workingjubilee in [#5812](github.com/gfx-
```

Comment:
issue: As with <https://github.com/gfx-rs/wgpu/pull/5850#discussion_r1647830103>, this is missing the protocol specifier (i.e., `https://). 😅